package com.github.connector.exception;

public class UserNotFoundException extends RuntimeException{

    public UserNotFoundException(String msg) { super(msg); }
}
